traits/src/nft.rs

```diff
@@ -27,3 +27,26 @@ pub trait NFT<AccountId> {
 	/// Transfer the given token ID from one account to another.
 	fn transfer(from: &AccountId, to: &AccountId, token: (Self::ClassId, Self::TokenId)) -> DispatchResult;
 }
+
+// This trait provides interface to manage NFTs
+#[allow(clippy::upper_case_acronyms)]
+pub trait ManageNFT<AccountId, CID, Attributes> {
```

Comment:
make it `NftMinter: NFT` and `NftBurner: NFT` so no need to duplicate `type ClassId`

Reply:
Split this into seperate traits